#' Obtain the Mair_benchmark data
#'
#' Obtain the targeted CITE-seq data from Mair \emph{et al.} (2020).
#'
#'@details
#'CITE-seq with targeted RNA-seq of PBMCs from 3 healthy donors carrying a HLA
#'subtype that allows identification of the Epstein-Barr Virus (EBV)-specific
#'CD8+ T-cells, comprising:
#'\itemize{
#' \item{EBV CD8+ T-cells enriched by FACs}
#' \item{Live CD45+ cells sorted from the PBMCs}
#'}
#'A corresponding flow cytometry antibody panel was used for comparison.
#'Whole transcriptome scRNA-seq was also performed for one donor.
#'
#' T-cells prior to and after stimulation.
#'
#'
#'Mair \emph{et al.} benchmarked CITE-seq against conventional flow cytometry.
#'CITE-seq was performed using the BD Biosciences Rhapsody platform.
#'
#'A list of antibody staining dilutions is in Supplementary Table 4
#'of the original article.  A code repository is available (see below).
#'
#'This data set was downloaded from
#'\url{https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135325}
#'
#'@references
#'Mair, Florian, et al. "A targeted multi-omic analysis approach measures
#'protein expression and low-abundance transcripts on the single-cell level."
#'Cell reports 31.1 (2020): 107499.
#'\doi{https://doi.org/10.1016/j.celrep.2020.03.063}
#'@seealso
#'Code repository \url{https:// github.com/MairFlo/Targeted_transcriptomics}
#'The flow cytometry data has been deposited in \url{www.flowrepository.org}
#'under accession FR-FCM-Z266.
#'@export
Mair_benchmark <- function(){

}
